packages/node_modules/@webex/plugin-meetings/src/metrics/mqa-processor.js

```diff
@@ -0,0 +1,110 @@
+import bowser from 'bowser';
+
+import {
+  STATS,
+  MQA_STATS
+} from '../constants';
+
+
+/**
+ * @description MQAProcessor handles interval data for MQA
+ * @export
+ * @class MQAProcessor
+ */
+class MQAProcessor {
+  /**
+     * @constructor
+     * @public
+     * @memberof Meetings
+     */
+  constructor() {
+    this.data = {
+      videoReceive: [],
+      audioTransmit: [],
+      audioReceive: [],
+      videoTransmit: []
+    };
+    this.intervalNumber = 1;
+  }
+
+  /**
+   * @param {String} id
+   * @param {Array} interval - a slice of metrics history
+   * @returns {undefined}
+   */
+  process(id, interval) {
+    let rtcCandidatePair, rtcOutVideo, rtpOutVideo; // TODO: , rtcInVideo, rtpInVideo, rtcOutAudio, rtcInAudio, rtpInAudio, rtpOutAudio; // TODO:
```

Comment:
Is there a JIRA for this TODO?

Reply:
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-122143